TetrisEngine.java

```diff
@@ -0,0 +1,134 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+
+public class TetrisEngine {
+	private static final String INPUT_FILE = "input.txt";
+	private static final String OUTPUT_FILE = "output.txt";
+	//private static final int GRID_SIZE = 100;
+	
+	private static HashMap<Character, TetrisPiece> PieceMap = new HashMap<Character, TetrisPiece>();

```

Comment:
```suggestion
	private static HashMap<Character, TetrisPiece> PieceMap = new HashMap<>();
```
You don't have to specify generic types on the right hand side, the compiler will infer them